internal/staging/util.go

```diff
@@ -24,28 +26,33 @@ func (c *controller) getDeployConfiguration(queue *s2hv1.Queue) *s2hv1.ConfigDep
 		return &s2hv1.ConfigDeploy{}
 	}
 
+	var configDeploy *s2hv1.ConfigDeploy
```

Comment:
Should we initiate like
```
configDeploy := &s2hv1.ConfigDeploy{}
```

and in return, we can remove these lines
```
if configDeploy == nil {
	configDeploy = &s2hv1.ConfigDeploy{}
}
```

Reply:
Okay